website/content/docs/developers/topologies.md

```diff
@@ -5,6 +5,23 @@ title: Writing a Topology
 A topology specifies components like spouts and bolts, as well as the relation
 between components and proper configurations.
 
+### Install Heron APIs for development
+
+Before getting started writing a topology, you need to install Heron API and 
+import its library to your own topology project.
+
+* Go to the releases page for Heron and download the Heron API installation 
+script for your platform. The name of the script for Mac OS X (`darwin`), 
+for example, would be named `heron-api-install-{{% heronVersion %}}-darwin.sh`.
+
+* Once you've downloaded, run the installation script with the `--user` flag set.
+
+* After successful installation, import `~/.heronapi/heron-storm.jar` to your 
```

Comment:
We should also refer users to the published artifacts in maven and include a pom.xml snippet for referencing artifacts that way:
https://search.maven.org/#search|ga|1|heron

See related maven tickets:
https://github.com/twitter/heron/issues?utf8=%E2%9C%93&q=is%3Aissue+publish+maven